src/screens/Exchange/ExchangeInfo.js

```diff
@@ -0,0 +1,266 @@
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+import * as React from 'react';
+import { FlatList, View, RefreshControl } from 'react-native';
+import { SDK_PROVIDER } from 'react-native-dotenv';
+import type { NavigationScreenProp } from 'react-navigation';
+import { connect } from 'react-redux';
+import styled from 'styled-components/native';
+import { format as formatDate } from 'date-fns';
+
+import { Container, ScrollWrapper } from 'components/Layout';
+import Header from 'components/Header';
+import { BoldText, BaseText } from 'components/Typography';
+import ListItemWithImage from 'components/ListItem/ListItemWithImage';
+import CollapsibleListItem from 'components/ListItem/CollapsibleListItem';
+import Separator from 'components/Separator';
+import { baseColors, fontSizes, spacing } from 'utils/variables';
+import { fetchTransactionsHistoryAction } from 'actions/historyActions';
+import {
+  PROVIDER_CHANGELLY,
+  PROVIDER_SHAPESHIFT,
+  PROVIDER_UNISWAP,
+  PROVIDER_ZEROX,
+} from 'constants/exchangeConstants';
+import type { Assets } from 'models/Asset';
+import type { Allowance } from 'models/Offer';
+
+type Props = {
+  navigation: NavigationScreenProp<*>,
+  assets: Assets,
+  exchangeAllowances: Allowance[],
+  fetchTransactionsHistory: Function,
+};
+
+type State = {
+  openCollapseKey: string,
+};
+
+const SectionTitle = styled(BoldText)`
+  font-size: ${fontSizes.medium}px;
+  margin: 16px;
+  margin-bottom: 10px;
+`;
+
+const ProviderItem = styled.View`
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-vertical: ${spacing.large}px;
+  padding-left: 44px;
+`;
+
+const ProviderName = styled(BaseText)`
+  font-size: ${fontSizes.small}px;
+`;
+
+const ProviderStatus = styled(BaseText)`
+  font-size: ${fontSizes.extraSmall}px;
+  color: ${props => props.isPending ? baseColors.darkGray : baseColors.jadeGreen}
+`;
+
+const DisconnectButton = styled.TouchableOpacity`
+  padding: 10px;
+  margin-right: -10px;
+`;
+
+const DisconnectButtonLabel = styled(BaseText)`
+  font-size: ${fontSizes.extraSmall}px;
+  color: ${baseColors.burningFire}
+`;
+
+const genericToken = require('assets/images/tokens/genericToken.png');
+
+const getProviderDisplayName = (provider: string) => {
+  switch (provider) {
+    case PROVIDER_SHAPESHIFT:
+      return 'ShapeShift';
+    case PROVIDER_UNISWAP:
+      return 'Uniswap';
+    case PROVIDER_ZEROX:
+      return '0x';
+    case PROVIDER_CHANGELLY:
+      return 'Changelly';
+    default:
+      return 'Unknown';
+  }
+};
+const dummyExchanges = [
```

Comment:
Shall we rename this?

Reply:
It's just a placeholder info for styling. I've already removed in my following commit